app/pages/Accounts/AccountReport/util.ts

```diff
@@ -140,27 +114,25 @@ function parseTransaction(
     return exportedFields.map((field) => fieldValues[getName(field)]);
 }
 
-function showingShieldedTransfers(filters: FilterOption[]) {
-    return filters.some(
-        (filter) => filter.key === TransactionKindString.EncryptedAmountTransfer
+function showingShieldedTransfers(filters: TransactionFilter) {
+    return getActiveBooleanFilters(filters).includes(
+        TransactionKindString.EncryptedAmountTransfer
     );
 }
 
 export async function containsEncrypted(
     account: Account,
-    filterOptions: FilterOption[],
-    fromTime?: Date,
-    toTime?: Date
+    filters: TransactionFilter
 ) {
-    if (
-        !showingShieldedTransfers(filterOptions) ||
-        !hasEncryptedBalance(account)
-    ) {
+    if (!showingShieldedTransfers(filters) || !hasEncryptedBalance(account)) {
         return false;
     }
 
-    const fromBlockTime = fromTime ? fromTime.getTime() : Date.now();
-    const toBlockTime = toTime ? toTime.getTime() : 0;
+    const { fromDate, toDate } = filters;
+
+    const fromBlockTime = fromDate ? new Date(fromDate).getTime() : Date.now();
+    const toBlockTime = toDate ? new Date(toDate).getTime() : 0;
```

Comment:
The defaults here should be split, actually, right?

Reply:
I would also assume so. Probably assumed it was correct prior to the change 😄